.circleci/config.yml

```diff
@@ -7,10 +7,25 @@ jobs:
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
+      - restore_cache:
+          keys: 
+            - carthage-cache-{{ checksum "Cartfile.resolved" }}
       - run:
           name: Carthage Bootstrap
-          command: carthage bootstrap
-      - run: bundle install
+          command: |
+              carthage bootstrap --cache-builds
+      - save_cache:
```

Comment:
let's separate these a bit with line breaks before and after each restore-run-save block